bin/templates/default.tmpl

```diff
@@ -1,6 +1,6 @@
 import angular from 'angular';
 
-var <%= componentName %> = angular.module('<%= componentName %>', []);
```

Comment:
Had a chat with @theefer and @jamesgorrie. It probably does work, but is more about where you want to be consistent (with the view, vs other objects such as controllers that are called from JS). Objects that are called from JS should probably be camel case since dashes won't work for that.

Since we chose camel case for directives, maybe go for that here as well.